docusaurus.config.ts

```diff
@@ -20,6 +20,12 @@ function injectTypeDocSidebar(items) {
             link: { type: "doc", id: "api/web/index" },
             items: require("./docs/api/web/typedoc-sidebar.cjs"),
           },
+          {
+            type: "category",
+            label: "Server SDK for JS",
```

Comment:
I'm open for renaming this label (it is visible in left sidebar)

Reply:
seems fine tbh